### ISSUE TITLE
Specify approach for resource meta-data

### ISSUE THREAD

**Issue by justinwb** [solid/specification#102]
We need to specify the approach for meta-data associated with a given resource (document or container), understanding that there is already a variety of metadata to account for today, and a need to be able to support new forms of metadata in a standardized way over time.

Examples of meta-data include (but are not limited to) wac permissions, shape validations, and audit history. Meta-data should apply to both RDF and non-RDF resources. Meta-data should be fully portable. Meta-data should be tied to the lifecycle of the resource. Meta-data should have a coherent mechanism for authorized access.

There are several issues that are directly or tangentially related to this, including #63, #65, #58. Creating this one issue as an overarching place to track them across the board.

**Comment by kjetilk** [comment]
Yeah, I can see the importance of this. I just don't think we are able to spec this for the Dec 19 release beyond what's in #63. So, I'm taken #63 to be about specing the `.meta` as it is currently implemented properly to be an existing feature, whereas the more general view that you propose that addresses more generic use cases, like linking to shape constraints and stuff, to be a new feature that we need to deal with later.

**Comment by justinwb** [comment]
> So, I'm taken #63 to be about specing the `.meta` as it is currently implemented properly to be an existing feature, whereas the more general view that you propose that addresses more generic use cases, like linking to shape constraints and stuff, to be a new feature that we need to deal with later.

This is so fundamental that I don't think we can afford not to look at it now. This was moved to Jun 19th as I'm typing this. Feels like we're kicking the can down the road on something that is going to be much more difficult to walk back later, and that shouldn't be overly complex to agree on now. Can we compromise on the March milestone? I believe we can submit a candidate proposal by the end of the year.

**Comment by kjetilk** [comment]
What?! I edited your comment, @justinwb ... That was not the intention, I didn't know I could... Anyway, We can make that compromise.

**Comment by justinwb** [comment]
> What?! I edited your comment, @justinwb ... That was not the intention, I didn't know I could... Anyway, We can make that compromise.

lol i don't think i've seen that before - but that sounds good to me!
